docs/dev-process/pull-request-process.md

```diff
@@ -0,0 +1,51 @@
+# Pull Request Process
+
+Requirements for a pull request to be accepted into PowerShell
+* Writing tests
+* Writing documentation
+
+## Pull Request Workflow
+
+1. The PR *author* creates a pull request from a fork.
+1. The *author* ensures that their pull request passes the [CI system][ci-system] build.
+  - If the build fails, a [Repository Maintainer][repository-maintainer] adds the `Review - waiting on author` label to the pull request. 
+  The *author* can then continue to update the pull request until the build passes.
+3. If the *author* knows whom should participate in the review, they should add them otherwise they can add the recommended *reviewers*.
+1. Once the build passes, if there is not sufficient review, the *maintainer* adds the `Review - needed` label.
+1. An [Area Expert][area-expert] should also review the pull request.
+  - If the *author* does not meet the *reviewer*'s standards, the *reviewer* makes comments. A *maintainer* then removes the `Review - needed` label and adds the `Review - waiting on author` label. The *author* must address the comments and repeat from step 2.
+  - If the *author* meets the *reviewer*'s standards, the *reviewer* approves the PR. A maintainer then removes the `need review` label.
+6. Once the code review is completed, a *maintainer* merges the pull request after one business day to allow for additional critical feedback.
+
+### Roles and Responsibilities
```

Comment:
There is a similar section in https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---code-review. For example, both mention 'author should drive the PR'. Maybe we should somehow link these two, or maybe we move this to the 'CONTRIBUTING.md'?